README.md

```diff
@@ -85,932 +119,667 @@ Unlike many authorization systems where a service controls access to resources i
   subgraph msgGraph [ ]
-    msg/send --> msg/*
-    msg/receive --> msg/*
+    /msg/send --> /msg
+    /msg/receive --> /msg
   end
 
-  crud/* --> *
+  /crud --> /
   subgraph crudGraph [ ]
-    crud/read --> crud/*
-    crud/mutate --> crud/*
+    /crud/read --> /crud
+    /crud/mutate --> /crud
 
     subgraph mutationGraph [ ]
-        crud/create --> crud/mutate
-        crud/update --> crud/mutate
-        crud/destroy --> crud/mutate
+        /crud/mutate/create --> /crud/mutate
+        /crud/mutate/update --> /crud/mutate
+        /crud/mutate/destroy --> /crud/mutate
     end
   end
 
-  ... --> *
+  ... --> /
 ```
 
+### Reserved Commands
 
-### 5.2.1 Bottom
-
-In concept there is a "bottom" ability ("none" or "void"), but it is not possible to represent in an ability. As it is merely the absence of any ability, it is not possible to construct a capability with a bottom ability.
+#### `/ucan` Namespace
 
-# 6. Validation
+The `/ucan` Command namespace MUST be reserved. This MUST include any ability string matching the regex `^ucan\/.*`. This is important for keeping a space for community-blessed Commands in the future, such as standard library Commands, such as [Revocation].
 
-Each capability has its own semantics, which needs to be interpretable by the target resource handler. Therefore, a validator SHOULD NOT reject UCANs with resources that it does not know how to interpret.
+## Attenuation
 
-If any of the following criteria are not met, the UCAN MUST be considered invalid.
+Attenuation is the process of constraining the capabilities in a delegation chain. Each direct delegation MUST either directly restate or attenuate (diminish) its capabilities.
 
-## 6.1 Time
+# Token Resolution
 
-A UCAN's time bounds MUST NOT be considered valid if the current system time is before the `nbf` field or after the `exp` field. This is called "ambient time validity."
+Token resolution is transport specific. The exact format is left to the relevant UCAN transport specification. At minimum, such a specification MUST define at least the following:
 
-All proofs MUST contain time bounds equal to or broader than the UCAN being delegated. If the proof expires before the outer UCAN — or starts after it — the reader MUST treat the UCAN as invalid. Delegation inside of the time bound is called "timely delegation." These conditions MUST hold even if the current wall clock time is inside of incorrectly delegated bounds. 
+1. Request protocol
+2. Response protocol
+3. Collections format
 
-A UCAN is valid inclusive from the `nbf` time and until the `exp` field. If the current time is outside of these bounds, the UCAN MUST be considered invalid. When setting these bounds, a delegator or invoker SHOULD account for expected clock drift. Use of time bounds this way is called "timely invocation."
+Note that if an instance cannot dereference a CID at runtime, the UCAN MUST fail validation. This is consistent with the [constructive semantics] of UCAN.
```

Comment:
Note that this is the first mention of a CID in the entire high-level spec, without a normref or definition.

Editorial nitpicking aside, I think it might be clearer to refer to them as `link`s in this and other section (which, in IPLD, are effectively just IPFS CIDs) rather than refer to them as CIDs, because what matters is that they are identifiers they can't be defererenced, not anything about their nature as _content_ identifiers or multiformats, etc etc.  (I.e., I can imagine an RDF/JSON-LD version of this where the same link-dereferenceability rule applies to DID URLs, or HTTPS URLS, or some other kinds of dereferenceable URIs...)

Reply:
For example:
```suggestion
Note that if a UCAN includes links in the form of [IPFS CIDs][CID] to subgraphs, and a parsing implementation cannot expand the full graph because one or more of these links cannot be dereferenced at runtime, then the UCAN MUST fail validation. This is consistent with the [constructive semantics] of UCAN.
```